### ISSUE TITLE
Incompatibility with Phosphor

### ISSUE THREAD

**Issue by mxnmnm** [LXGaming/MixinBootstrap#52]
- MixinBootstrap version: 1.1.0
- Forge version: 14.23.5.2860
- Minecraft version: 1.12.2
- Java version: Java 8 
- Mods: Phosphor

**Issue Description**
Version 1.1.0 is incompatible with phosphor
Log: https://gist.github.com/mxnmnm/fb00b9c070e6eda4da0a5c46aed5d9bd

**Comment by LXGaming** [comment]
Could you please provide a debug.log

**Comment by mxnmnm** [comment]
Only the log is created, I don't see a debug.log

Is there a way to force a debug.log to be created?

**Comment by mxnmnm** [comment]
So it seems the mixin 0.7-0.8 compat mod managed to make it run just fine, although it would probably be ideal to just fix the issue. So tell me what I need to do to find the debug.log (I might've needed to do this in the past and I think it involved stuff like running minecraft in a weird way it was a long time ago lol)

**Comment by LXGaming** [comment]
What launcher are you using?

**Comment by mxnmnm** [comment]
GDLauncher

**Comment by LXGaming** [comment]
The debug.log should be in the logs folder, I checked and can see that it exists when running the Minecraft or MultiMC Launcher so I'm assuming GDLauncher might be overriding the default logging behavior.

I've managed to reproduce the error, It looks like changes were made to Mixin 0.8.3-0.8.4 which causes issue with phosphor, There is nothing I can do about this, MixinBootstrap simply adds the Mixin library to Minecraft I can't change Mixin to make it compatible with mods.

Solutions:
- Remove MixinBootstrap, phosphor includes its own version of Mixin which it is compatible with.
- Update phosphor to support a newer version of Mixin, Although it looks like jellysquid3 nuked the official repo [jellysquid3/phosphor-forge](https://github.com/jellysquid3/phosphor-forge) :(
